From 22d6a7ac30d10eba804cb59672d8bb8dd1623998 Mon Sep 17 00:00:00 2001
From: WyattBlue <wyattblue@auto-editor.com>
Date: Mon, 4 Aug 2025 19:28:45 -0400
Subject: [PATCH 12/13] Remove unused swr functions

---
 include/libswresample/swresample.pxd | 35 ----------------------------
 1 file changed, 35 deletions(-)

diff --git a/include/libswresample/swresample.pxd b/include/libswresample/swresample.pxd
index 65b8314..5bd8a86 100644
--- a/include/libswresample/swresample.pxd
+++ b/include/libswresample/swresample.pxd
@@ -1,39 +1,4 @@
-from libc.stdint cimport int64_t, uint8_t
-
-
 cdef extern from "libswresample/swresample.h" nogil:
-
     cdef int   swresample_version()
     cdef char* swresample_configuration()
     cdef char* swresample_license()
-
-    cdef struct SwrContext:
-        pass
-
-    cdef SwrContext* swr_alloc_set_opts(
-        SwrContext *ctx,
-        int64_t out_ch_layout,
-        AVSampleFormat out_sample_fmt,
-        int out_sample_rate,
-        int64_t in_ch_layout,
-        AVSampleFormat in_sample_fmt,
-        int in_sample_rate,
-        int log_offset,
-        void *log_ctx  # logging context, can be NULL
-    )
-
-    cdef int swr_convert(
-        SwrContext *ctx,
-        uint8_t ** out_buffer,
-        int out_count,
-        uint8_t **in_buffer,
-        int in_count
-    )
-    # Gets the delay the next input sample will
-    # experience relative to the next output sample.
-    cdef int64_t swr_get_delay(SwrContext *s, int64_t base)
-
-    cdef SwrContext* swr_alloc()
-    cdef int swr_init(SwrContext* ctx)
-    cdef void swr_free(SwrContext **ctx)
-    cdef void swr_close(SwrContext *ctx)
-- 
2.49.0

