diff --git a/setup.py b/setup.py
index af8111e..6b81f8c 100644
--- a/setup.py
+++ b/setup.py
@@ -148,7 +148,7 @@ loudnorm_extension = Extension(
     include_dirs=[f"{IMPORT_NAME}/filter"] + extension_extra["include_dirs"],
     libraries=extension_extra["libraries"],
     library_dirs=extension_extra["library_dirs"],
-    define_macros=[("Py_LIMITED_API", 0x030B0000)],
+    # define_macros=[("Py_LIMITED_API", 0x030B0000)],
     py_limited_api=py_limited_api,
 )
 
@@ -194,7 +194,7 @@ for dirname, dirnames, filenames in os.walk(IMPORT_NAME):
                 libraries=extension_extra["libraries"],
                 library_dirs=extension_extra["library_dirs"],
                 sources=[pyx_path],
-                define_macros=[("Py_LIMITED_API", 0x030B0000)],
+                # define_macros=[("Py_LIMITED_API", 0x030B0000)],
                 py_limited_api=py_limited_api,
             ),
             compiler_directives=compiler_directives,

