diff --git a/tests/test_codec_context.py b/tests/test_codec_context.py
index ee9193c..43e827a 100644
--- a/tests/test_codec_context.py
+++ b/tests/test_codec_context.py
@@ -217,7 +217,7 @@ class TestEncoding(TestCase):
                 self.assertEqual(frame.format.name, pix_fmt)
 
     def test_encoding_h264(self):
-        self.video_encoding("libx264", {"crf": "19"})
+        self.video_encoding("libopenh264", {"crf": "19"})
 
     def test_encoding_mpeg4(self):
         self.video_encoding("mpeg4")
diff --git a/tests/test_encode.py b/tests/test_encode.py
index a293cbf..b3da294 100644
--- a/tests/test_encode.py
+++ b/tests/test_encode.py
@@ -149,7 +149,7 @@ class TestBasicVideoEncoding(TestCase):
         path = self.sandboxed("video_with_pts.mov")
 
         with av.open(path, "w") as output:
-            stream = output.add_stream("libx264", 24)
+            stream = output.add_stream("libopenh264", 24)
             self.assertIn(stream, output.streams.video)
             stream.width = WIDTH
             stream.height = HEIGHT
