From 2186a380ef7d7b0dcc3427d0208f31edccc821ea Mon Sep 17 00:00:00 2001
From: WyattBlue <wyattblue@auto-editor.com>
Date: Fri, 4 Jul 2025 15:48:19 -0400
Subject: [PATCH] Remove checks in setup.py

---
 setup.py | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/setup.py b/setup.py
index 16ca7f59b..a96e0a019 100644
--- a/setup.py
+++ b/setup.py
@@ -7,20 +7,6 @@
 import subprocess
 import sys
 
-if platform.system() == "Darwin":
-    major_version = int(platform.mac_ver()[0].split(".")[0])
-    if major_version < 12:
-        raise OSError("You are using an EOL, unsupported, and out-of-date OS.")
-
-
-def is_virtualenv():
-    return sys.base_prefix != sys.prefix
-
-
-if not (os.getenv("GITHUB_ACTIONS") == "true" or is_virtualenv()):
-    raise ValueError("You are not using a virtual environment")
-
-
 from Cython.Build import cythonize
 from Cython.Compiler.AutoDocTransforms import EmbedSignature
 from setuptools import Extension, find_packages, setup
